app/routes/events/live.js

```diff
@@ -5,5 +5,19 @@ const { Route } = Ember;
 export default Route.extend({
   titleToken() {
     return this.i18n.t('Live');
+  },
+  model() {
+    return [{
+      name     : 'Event_Name2',
+      startAt  : 'March 20, 2015 - 05:30 PM',
+      endAt    : 'March 20, 2016 - 05:30 PM',
```

Comment:
@anu-007 The first implementation of ui-table was just to get started with, I have opened #161 to address this, it will be handled in it. Thanks.